charts/celeborn/values.yaml

```diff
@@ -19,41 +19,52 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-# we provide a default celeborn image, you can also replace to your image
-# TODO rebuild celeborn official image
+# -- String to override the default generated name
+nameOverride: ""
+
+# -- String to override the default generated fullname
+fullnameOverride: ""
+
+# Specifies the Celeborn image to use
 image:
+  # -- Image repository
   repository: aliyunemr/remote-shuffle-service
-  pullPolicy: Always
+  # -- Image tag
   tag: 0.1.1-6badd20
+  # -- Image pull policy
+  pullPolicy: Always
 
+# -- Image pull secrets for private image registry
 imagePullSecrets: []
 
-# master replicas should not less than 3
+# -- Specifies the number of Celeborn master replicas to deploy, master replicas should not less than 3
 masterReplicas: 3
-# worker replicas set on demand, should less than node number
+# -- Specifies the number of Celeborn worker replicas to deploy, should less than node number
 workerReplicas: 5
 
-hostNetwork: false
-dnsPolicy: ClusterFirst
+service:
+  # -- Specifies service type
+  type: ClusterIP
+  # -- Specifies service port
+  port: 9097
 
-securityContext:
-  runAsUser: 10006
-  runAsGroup: 10006
-  fsGroup: 10006
+cluster:
+  # -- Specifies Kubernetes cluster name
+  name: cluster
 
-# Current Celeborn support followings volume type:
-# - emptyDir
-# - hostPath
-# Note:
-# hostPath only works in hostPath type using to set `volumes hostPath path`
-# Celeborn Master will pick first volumes for store raft log
-# diskType only works in Celeborn Worker with hostPath type to manifest local disk type
+# -- Specifies Celeborn volumes.
+# Currently supported volume types are `emptyDir` and `hostPath`. <br>
+# Note that `hostPath` only works in hostPath type using to set `volumes hostPath path`.
+# Celeborn Master will pick first volumes for store raft log. <br>
+# `diskType` only works in Celeborn Worker with hostPath type to manifest local disk type.
 volumes:
+  # @ignore
```

Comment:
what does `@ignore` use for? and ignore what?

Reply:
> what does `@ignore` use for? and ignore what?

@pan3793 It's used by helm-docs to generate values section in helm chart README.md. Without `@ignore` annotation, the values of volumes will be rendered in README file, which is too long and will make it hard to read.

Reply:
it seems that the most important value of the README template is auto-generating to the table of parameters, I would argue this might not be much more helpful than `value.yaml` built-in comments, but introduces complexity.

Reply:
> it seems that the most important value of the README template is auto-generating to the table of parameters, I would argue this might not be much more helpful than `value.yaml` built-in comments, but introduces complexity.

@pan3793 It definitely introduces complexity, but it can also make the README doc up-to-date  with the help of github actions. Some people are more willing to read README.md to learn about how to install, update and configure the helm chat and I am willing to update the helm usage docs in the  future. `@ignore` annotation is somewhat confusing, and I will try to fix that.